### PR TITLE
Feature FLU: Aibo Bioloid Qrio

### PR DESCRIPTION
**Pull request by BenjaminDeleze**
**Description**

### Robots
- [x] Aibo
- [x] Bioloid
- [x] Qrio

### Worlds

- [x] aibo.wbt
- [x] bioloid_dog.wbt
- [x] qrio.wbt

### CODE REVIEW

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Looks good except:
- `QRIO.proto`: x-axis is not forward.
- `aibo_ers7.wbt` and `qrio.wbt` worlds were not checked in #3073 (I have just checked them).

File: projects/robots/sony/qrio/protos/QRIO.proto [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # QRIO ("Quest for cuRIOsity", originally named Sony Dream Robot or SDR) is a bipedal humanoid entertainment robot developed and marketed by Sony.
 
 PROTO QRIO [
-  field SFVec3f    translation     0 0 0   # Is `Transform.translation`.
+  field SFVec3f    translation     0 0 0.42   # Is `Transform.translation`.
```
Comment (lukicdarkoo) [inline projects/robots/sony/qrio/protos/QRIO.proto:8]: Not aligned

Comment (lukicdarkoo) [reply projects/robots/sony/qrio/protos/QRIO.proto:8]: Also, it should be changed in `qrio.md` as well

**Comment by BenjaminDeleze** [comment]
I think I addressed all coments

File: projects/robots/robotis/bioloid/protos/BioloidDog.proto [hunk 2]
```diff
@@ -35,7 +35,8 @@ PROTO BioloidDog [
         range 999
       }
       DEF CM5_TRANS Transform {
-        rotation 0 1 0 1.5708
+        rotation 0 0 1 1.5708
+        translation 0 0 0
```
Comment (lukicdarkoo) [inline projects/robots/robotis/bioloid/protos/BioloidDog.proto:39]:
```suggestion
```
I missed a tiny detail :/

**Review by lukicdarkoo — APPROVED** [review]
Thank you!

**Pull request merged**